src/controllers/UserController.js

```diff
@@ -184,6 +184,43 @@ class UserController {
       return HelperMethods.serverError(res);
     }
   }
+
+  /**
+  * Verify a user's email
+  * Route: POST: /auth/verify_email
+  * @param {object} req - HTTP Request object
+  * @param {object} res - HTTP Response object
+  * @return {res} res - HTTP Response object
+  * @memberof UserController
+ */
+  static async updatedUser(req, res) {
+    const token = req.headers['x-access-token'];
+    const { role, email } = req.body;
+    const verifiedToken = await Authentication.verifyToken(token);
+    try {
+      if (verifiedToken.role !== 'Super Administrator') {
+        return HelperMethods.clientError(res, 'Only a super admin'
```

Comment:
Base on my explanation on line 197, line 202 will be `payload.id !== super Administrator`.